cmd/rm-main.go

```diff
@@ -121,14 +132,24 @@ func checkRmSyntax(ctx *cli.Context) {
 	isForce := ctx.Bool("force")
 	isRecursive := ctx.Bool("recursive")
 	isStdin := ctx.Bool("stdin")
-
+	isNamespace := ctx.Bool("namespace")
+	isNamespaceRemoval := false
+	for _, url := range ctx.Args() {
+		if isHostName(url) {
```

Comment:
This method does not work for removing a directory with non-aliases.  For example `mc rm mylocaldir` would fail.  Basically we need to differentiate alias and local dir

Reply:
changed this to incorporate feedback